src/CommunityToolkit.Maui/Behaviors/MaskedBehavior.shared.cs

```diff
@@ -35,127 +36,135 @@ public class MaskedBehavior : BaseBehavior<InputView>
 	/// <summary>
 	/// The placeholder character for when no input has been given yet. This is a bindable property.
 	/// </summary>
-	public char UnMaskedCharacter
+	public char UnmaskedCharacter
 	{
-		get => (char)GetValue(UnMaskedCharacterProperty);
-		set => SetValue(UnMaskedCharacterProperty, value);
+		get => (char)GetValue(UnmaskedCharacterProperty);
+		set => SetValue(UnmaskedCharacterProperty, value);
 	}
 
-	static void OnMaskPropertyChanged(BindableObject bindable, object oldValue, object newValue)
-		=> ((MaskedBehavior)bindable).SetPositions();
-
-	static void OnUnMaskedCharacterPropertyChanged(BindableObject bindable, object oldValue, object newValue)
-		=> ((MaskedBehavior)bindable).OnMaskChanged();
-
 	/// <inheritdoc />
-	protected override void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
+	protected override async void OnViewPropertyChanged(InputView sender, PropertyChangedEventArgs e)
 	{
 		base.OnViewPropertyChanged(sender, e);
 
 		if (e.PropertyName == InputView.TextProperty.PropertyName)
 		{
-			OnTextPropertyChanged();
+			await OnTextPropertyChanged();
```

Comment:
Can we and should we use Cancellation token here?

Reply:
Absolutely! Good catch 💯

Reply:
I added the `CancellationToken token = default` as a parameter, but then realized that there's no way to pass in a token from .NET MAUI. I.e. These methods are (currently) only called from `PropertyChanged` which doesn't provide a token. 

But, I still think it's a good idea to have a `CancellationToken` as a parameter so that these methods can be leveraged by future code that maybe can supply a token 👍